touchforms/backend/xformplayer.py

```diff
@@ -445,7 +445,7 @@ def _parse_repeat_juncture(self, event):
         ro = r.getRepeatOptions()
 
         event.update(get_caption(r))
-        event['main-header'] = ro.header
```

Comment:
`main-header` is a terrible name when you're accessing an attribute in javascript